gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixTaskDriver.java

```diff
@@ -304,14 +304,14 @@ public ZNRecord update(ZNRecord currentData) {
         if (currentData != null) {
           // Only update target state for non-completed workflows
           String finishTime = currentData.getSimpleField(WorkflowContext.FINISH_TIME);
-          if (finishTime == null || finishTime.equals(WorkflowContext.UNFINISHED)) {
+          if (finishTime == null || Integer.valueOf(finishTime).equals(WorkflowContext.UNFINISHED)) {
```

Comment:
did a different bugfix make it into this PR by accident?

Reply:
There was a find bugs error because of which build was failing. Had to fix. So added as the part of PR. Added the same in the description